web/src/show.php

```diff
@@ -94,7 +95,7 @@
         <?php foreach ($fields as $field => $title): ?>
             <?php if (isset($student[$field])): ?>
                 <?php foreach ($student[$field] as $index => $value): ?>
-                    <tr>
+                    <tr class="<?php out(chr(82 + $index)) ?>">
```

Comment:
The $index here corresponds to the sort order given by the semester. It's not the same order as the similar IDs, so you need to search for the ID in $similarIds to get the index that can be converted into a letter. (See line 104)